lisa/util/process.py

```diff
@@ -197,7 +197,7 @@ def wait_result(
                 f"execution time: {self._timer}, exit code: {self._result.exit_code}"
             )
 
-        if expected_exit_code:
+        if expected_exit_code is not None:
```

Comment:
when will it be None?

Reply:
when not set? current logic is wrong, when expected_exit_code  = 0, it never enters into if